packages/ensnode-sdk/package.json

```diff
@@ -1,27 +1,32 @@
 {
-  "name": "@ensnode/utils",
+  "name": "@ensnode/ensnode-sdk",
   "version": "0.27.0",
   "type": "module",
   "description": "A utility library for interacting with ENS data",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/namehash/ensnode.git",
-    "directory": "packages/ensnode-utils"
+    "directory": "packages/ensnode-sdk"
   },
-  "homepage": "https://github.com/namehash/ensnode/tree/main/packages/ensnode-utils",
+  "homepage": "https://github.com/namehash/ensnode/tree/main/packages/ensnode-sdk",
   "files": [
     "dist"
   ],
   "exports": {
-    ".": "./src/index.ts"
+    ".": "./src/index.ts",
+    "./utils": "./src/utils/index.ts"
```

Comment:
Allowing clients to only import `utils` module by calling `import {} from "@ensnode/ensnode-sdk/utils";`.

Could be useful to express groups of exported members in a clear way. Thoughts?